### ISSUE TITLE
Nemoarchive Import - Script to convert metadata into JSON

### ISSUE THREAD

**Issue by adkinsrs** [IGS/gEAR#525]
Need a script to convert metadata from the imported JSON file into JSON for a typical gEAR upload. For some file formats (such as threetab) this is not necessary as the file is already present, but for others (like MEX) we need to rely on the metadata imported. Currently we also need to ensure the right metadata is being loaded into the nemoarchive portal.

Once this is done, we potentially could reuse scripts like "validate_metadata.cgi"

**Comment by adkinsrs** [comment]
This is complete and working but given that NeMO Archive does not have its updated API in place, I can only have it work with mock data

**Comment by adkinsrs** [comment]
Closing for now.  When the NeMO Archive API is finalized and we can avoid working with mock data I will create a new ticket
